src/lib/AnnotationControls.ts

```diff
@@ -141,11 +141,26 @@ export default class AnnotationControls {
     };
 
     /**
-     * Region comment button click handler
+     * Update the mode. If the mode is different from what is currently saved in state,
+     * then reset the current controls and apply the active state based on the provided mode.
+     * @param mode Annotation mode
```

Comment:
We decided to not use jsdoc in this file

Reply:
Gotcha thanks